repository-data/application/src/main/resources/hcm-config/namespaces/website/openingHours.yaml

```diff
@@ -19,64 +19,134 @@ definitions:
             item: ${cluster.id}.field
             jcr:primaryType: frontend:plugin
             plugin.class: org.hippoecm.frontend.service.render.ListViewPlugin
-          /day:
+          /monday:
             /cluster.options:
               jcr:primaryType: frontend:pluginconfig
-              source: /content/documents/administration/website/week-day
-            caption: Day
-            field: day
+            caption: Monday
+            field: monday
+            hint: e.g. '9am - 12pm, 1pm - 5pm' OR '9am-5pm (closed bank holidays)'
+              OR 'Training day'
```

Comment:
Also append ". Leave field blank to hide individual day." to hints to let user know how to hide a day

Reply:
added that